go/systests/team_reset_test.go

```diff
@@ -31,11 +32,33 @@ func divDebug(ctx *smuContext, fmt string, arg ...interface{}) {
 	ctx.log.Debug(div+" "+fmt+" "+div, arg...)
 }
 
-func readChatsWithError(team smuTeam, u *smuUser) ([]chat1.MessageUnboxed, error) {
+func readChatsWithError(team smuTeam, u *smuUser) (messages []chat1.MessageUnboxed, err error) {
```

Comment:
Fine with merging this, but is there plan to solve it a better way?

Reply:
I'd love to, I didn't want to include it in the PR because I thought I only have this problem locally (I guess my PC favors the "wrong" side of the race), but then we started seeing them in CI so we might as well have this for now.